usr/sbin/p11sak/p11sak.c

```diff
@@ -913,6 +924,81 @@ static void set_bool_attr_from_string(CK_ATTRIBUTE *attr, char attr_char)
     }
 }
 
+/**
+ * @brief Set the custom attrs from string object
+ * 
+ * @param attr The attribute array
+ * @param attr_char The string from the user
+ * @param count Index of current attribute in the attribute array
+ */
+static CK_RV set_custom_attrs_from_string(char *attr_char, CK_ATTRIBUTE *attr_array, CK_ULONG *count) 
+{
+    CK_RV rc = CKR_OK;
+    char *attr = NULL;
+    char *separator = ",";
+
+    // iterate over the single custom attributes
+    attr = strtok(attr_char, separator);
+    while (attr != NULL && *count < 128)
```

Comment:
Is the 128 the size of the array? Please use a #define for this, so that we don't have the numeric constant 128 at many places. If we ever have to increase the array, we only want to update that single define then.